### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by notspiff**
takes care of the things hudo mentioned in pr (and some more stuff in that area), as well as a fix for tests.

### CODE REVIEW

File: project/cmake/modules/FindVAAPI.cmake [hunk 1]
```diff
@@ -11,19 +11,23 @@
 
 find_path(VAAPI_INCLUDE_DIRS va/va.h)
 
-find_library(VA_LIBRARY
-             NAMES va>=1.4.0)
+if(PKG_CONFIG_FOUND)
+  pkg_check_modules(VAAPI libva>=0.38 libva-x11 libva-glx)
```
Comment (FernetMenta) [inline project/cmake/modules/FindVAAPI.cmake]: glx is not required, we switched to egl

Comment (notspiff) [inline project/cmake/modules/FindVAAPI.cmake]: righty'o.

**Comment by fetzerch** [comment]
thanks, i'll squash the fixes into the existing commits. think we should soonish also get jenkins to build this to avoid things like the forgotten ".bin" (was on osx that time)

File: xbmc/interfaces/legacy/Doxyfile.in [hunk 2]
```diff
@@ -58,7 +58,7 @@ PROJECT_LOGO           =
 # entered, it will be relative to the location where doxygen was started. If
 # left blank the current directory will be used.
 
-OUTPUT_DIRECTORY       = ./generated
+OUTPUT_DIRECTORY       = @legacy_doxy_output_dir@
```
Comment (fetzerch) [inline xbmc/interfaces/legacy/Doxyfile.in:61]: @notspiff: is it safe to move the Doxyfile? Or better copy it to not break the other buildsystems?

Comment (akva2) [inline xbmc/interfaces/legacy/Doxyfile.in:61]: Didnt consider that. Copy.

Comment (fetzerch) [inline xbmc/interfaces/legacy/Doxyfile.in:61]: Thanks! Done.

**Pull request merged**